models/ClimateSensor.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azsamples:iotdd:ClimateSensor;1",
+  "@type": "Interface",
+  "displayName": "Climate Sensor",
+  "description": "Provides functionality to report temperature, humidity, Pressure",
+  "comment": "Requires temperature, hunidity and pressure sensors.",
+  "contents": [
+    {
+      "@type": "Property",
+      "displayName": "Update interval",
+      "description": "Interval of telemetry updates in seconds",
+      "name": "updateInterval",
+      "writable" : true,
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Telemetry",
+        "SemanticType/Temperature"
+      ],
+      "description": "Current temperature on the device",
+      "displayName": "Temperature",
+      "name": "temperature",
+      "schema": "double"
+    },
+    {
+      "@type": [
+        "Telemetry",
+        "SemanticType/Humidity"
```

Comment:
Semantic types are not prefixed with "SemanticType/" and I think that the Humidity semantic type requires a unit property.